### ISSUE TITLE
FutureWarning in genutil.minmax (uvcdat 2.12)

### ISSUE THREAD

**Issue by jypeter** [UV-CDAT/genutil#14]
Don't know if this is something for @doutriaux1 or @dnadeau4 

I have just triggered **a new warning** that I did not get in 2.10. I guess this is due to the new numpy version in 2.12 (**numpy 1.13.1** instead of 1.12.1)

```
(cdatm16) bash-4.1$ python
Python 2.7.13 | packaged by conda-forge | (default, May  2 2017, 12:48:11) 
[GCC 4.8.2 20140120 (Red Hat 4.8.2-15)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import cdms2, vcs, genutil
>>> f = cdms2.open(vcs.sample_data + '/clt.nc')
>>> clt = f('clt', time=slice(0,1), squeeze=1)
>>> f.close()
>>> genutil.minmax(clt)
/home/share/unix_files/cdat/miniconda2/envs/cdatm16/lib/python2.7/site-packages/numpy/ma/core.py:6385: MaskedArrayFutureWarning: In the future the default for ma.maximum.reduce will be axis=0, not the current None, to match np.maximum.reduce. Explicitly pass 0 or None to silence this warning.
  return self.reduce(a)
/home/share/unix_files/cdat/miniconda2/envs/cdatm16/lib/python2.7/site-packages/numpy/ma/core.py:6385: MaskedArrayFutureWarning: In the future the default for ma.minimum.reduce will be axis=0, not the current None, to match np.minimum.reduce. Explicitly pass 0 or None to silence this warning.
  return self.reduce(a)
(0.0, 100.0)
>>> 
```

**Comment by dnadeau4** [comment]
@jypeter 
I do not get this warning on my machine.  I might have fixed it with the python 3 conversion.
```python 
>>> import cdat_info
>>> import cdms2
>>> path=cdat_info.get_sampledata_path
>>> f=cdms2.open(path()+'/clt.nc')
>>> clt=f('clt', time=slice(0,1), squeeze=1)
>>>i mport genutil
>>> genutil.minmax(clt)
(0.0, 100.0)

>>> print cdms2.MV2.numpy.version.version
1.12.1
```

**Comment by doutriaux1** [comment]
@dnadeau4 are you using numpy 1.13?

**Comment by doutriaux1** [comment]
@dnadeau4 it's introduced by numpy 1.13, please update your conda env

**Comment by dnadeau4** [comment]
@doutriaux1 I am not ready to update to 1.13.  Too many issues in that version.

**Comment by jypeter** [comment]
Have you guys seen my Friday issue about 2.12, cmor, and other packages?
-- 
JYP @ SGS 7

**Comment by doutriaux1** [comment]
@jypeter yes, it is due to cmor being stuck with hdf5 1.8.17 next version of cmor will be updated to hdf5 1.8.18

**Comment by doutriaux1** [comment]
@jypeter you can bring numpy 1.12 into uvcdat if you want, we build uvcdat against 1.11 1,12 and 1,13

```
conda install -f numpy=1.12
```

**Comment by dnadeau4** [comment]
@jypeter I did, I just release 3.2.6 and this will have to wait for 3.2.7.  Meanwhile, @doutriaux1 told me he will build a cmor package for you.

**Comment by doutriaux1** [comment]
@jypeter should be fixed by #15
